<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EqualizerMainWindow</class>
 <widget class="QMainWindow" name="EqualizerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI Equalizer - MVVM Architecture</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="equalizerWidget">
       <layout class="QVBoxLayout" name="eqLayout">
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="startStopButton">
            <property name="text">
             <string>Start Audio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="presetLayout">
          <item>
           <widget class="QLabel" name="presetLabel">
            <property name="text">
             <string>Preset:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="presetCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="eqGroup">
          <property name="title">
           <string>10-Band Equalizer</string>
          </property>
          <layout class="QHBoxLayout" name="eqMainLayout"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="ChatView" name="chatWidget">
       <property name="minimumHeight">
        <number>200</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatView</class>
   <extends>QWidget</extends>
   <header>ChatView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
